### PR TITLE
Verification user and react structure(routering)

### PR DESCRIPTION
**Pull request by onlyVone-zz**

### CODE REVIEW

File: src/main/java/com/softserve/actent/verification/service/SendEmail.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            MimeMessage message = new MimeMessage(session);
+            String to = email;
+
+            InternetAddress[] address = InternetAddress.parse(to, true);
+            message.setRecipients(Message.RecipientType.TO, address);
+            message.setFrom();
+            message.setSubject("Verification mail");
+            message.setSentDate(new Date());
+            message.setText("For being an activated user in Actent community and have all advantages of our service, " +
+                    "you need to click on the link below. \n \n" + confirmUrl + "\n\nWish you success!\nSincerely yourth, Actent's team");
```
Comment (md1guy) [inline src/main/java/com/softserve/actent/verification/service/SendEmail.java]: Correct all grammatical and lexical mistakes in this message.

File: src/main/java/com/softserve/actent/verification/service/SendEmail.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        try {
+            properties.load(SendEmail.class.getClassLoader().getResourceAsStream("mail.properties"));
+        } catch (IOException e) {
+            log.error("Cant download main.properties file");
+        }
+
+        Session session = Session.getInstance(properties, new javax.mail.Authenticator(){
+            @Override
+            protected PasswordAuthentication getPasswordAuthentication() {
+                return new PasswordAuthentication("onlyTwo2111@gmail.com", "qwert1234asd");
```
Comment (md1guy) [inline src/main/java/com/softserve/actent/verification/service/SendEmail.java]: Pure strings, especially passwords, should be declared as constants anywhere else.

File: src/main/java/com/softserve/actent/verification/service/Verification.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
+package com.softserve.actent.verification.service;
```
Comment (md1guy) [inline src/main/java/com/softserve/actent/verification/service/Verification.java:1]: Provide services as interfaces+their implementation.

File: src/main/java/com/softserve/actent/verification/service/SendEmail.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
+package com.softserve.actent.verification.service;
```
Comment (md1guy) [inline src/main/java/com/softserve/actent/verification/service/SendEmail.java:1]: https://github.com/lv-381-java/Actent/pull/174/files#r270595518

Comment (md1guy) [reply src/main/java/com/softserve/actent/verification/service/SendEmail.java:1]: .

File: src/main/java/com/softserve/actent/verification/service/SendEmail.java (hunk 2 above)
Comment (md1guy) [reply src/main/java/com/softserve/actent/verification/service/SendEmail.java]: .

File: src/main/java/com/softserve/actent/verification/service/SendEmail.java (hunk 1 above)
Comment (md1guy) [reply src/main/java/com/softserve/actent/verification/service/SendEmail.java]: .

File: src/main/java/com/softserve/actent/verification/service/Verification.java (hunk 3 above)
Comment (md1guy) [reply src/main/java/com/softserve/actent/verification/service/Verification.java:1]: .

**Review by octap16 — DISMISSED** [review]

**Review by nazarmavko — DISMISSED** [review]

**Review by octap16 — APPROVED** [review]

**Review by thebadpanda — APPROVED** [review]

**Review by irasht — APPROVED** [review]

**Pull request merged**